auth0_flutter/lib/src/web/extensions/web_exception_extensions.dart

```diff
@@ -1,15 +1,20 @@
-import 'dart:js_util';
+import 'dart:js_interop';
+import 'dart:js_interop_unsafe';
 import 'package:auth0_flutter_platform_interface/auth0_flutter_platform_interface.dart';
 
+@JS('Object.keys')
+external JSArray<JSString> keys(final JSObject o);
+
 extension WebExceptionExtension on WebException {
   static WebException fromJsObject(final Object jsException) {
-    final error = getProperty<String>(jsException, 'error');
-    final description = getProperty<String>(jsException, 'error_description');
-    final Map<String, dynamic> details = {};
+    final error = jsException.toJSBox.getProperty<JSString>('error'.toJS);
+    final description =
+        jsException.toJSBox.getProperty<JSString>('error_description'.toJS);
+    final Map<String, JSAny?> details = {};
 
-    objectKeys(jsException).forEach((final key) {
-      if (key == 'error' || key == 'error_description') return;
-      details[key as String] = getProperty<dynamic>(jsException, key);
+    keys(jsException.toJSBox).toDart.forEach((final key) {
```

Comment:
Small suggestion to make this easier to read. We should also not mix up the JS types with the Dart types.
Each Dart string should be manually converted to a JSString. While a JSString _might_ be equal to a String (it is one of the allowed primitives), this might not always be the case and could be JS backend dependent in the future.

```suggestion
    keys(jsException.toJSBox).toDart.forEach((final JSString key) {
       if (key == 'error'.toJS || key == 'error_description'.toJS) return;
       details[key.toDart] = jsException.toJSBox.getProperty<JSAny?>(key);
    });
```